lib/mirage.ml

```diff
@@ -1912,6 +1912,8 @@ module Project = struct
     "open Lwt.Infix\n\
      let return = Lwt.return\n\
      let run = OS.Main.run"
+  let packages = [package "mirage"]
```

Comment:
That's the dependency to compile `config.ml` (e.g. to build the object to dynlink to get the configuration)